tests/cypress/config/index.js

```diff
@@ -37,8 +37,10 @@ exports.getConfig = () => {
   let config;
   if (process.env.CYPRESS_TEST_MODE === "e2e") {
     config = fs.readFileSync(path.join(__dirname, "config.e2e.yaml"));
+    secretConfig = fs.readFileSync(SECRET_FILE_PATH);
   } else if (process.env.CYPRESS_TEST_MODE === "smoke") {
     config = fs.readFileSync(path.join(__dirname, "config.smoke.yaml"));
+    secretConfig = fs.readFileSync(SECRET_FILE_PATH);
   } else {
     config = fs.readFileSync(path.join(__dirname, "config.func.yaml"));
     secretConfig = fs.readFileSync(SECRET_FILE_PATH);

```

Comment:
Just do this once outside the if instead of repeating for every case?

Reply:
Good catch! I made the change.